codestodians/food-basket-front-end
# [Feature: #154880564] Create user profile component

#### What does this PR do?
- Creates a minimalistic component for having access to the profile of a `User`.

#### Description of Task to be completed?
- Disabling visibility of the `Search` and `Cart` components for certain pages.
- Creation of basic presentational and container profile components.
- Fixing of the everytime-update of the `auth` snapshot.

#### How should this be manually tested?
- Checkout to the `ft-user-profile-component-154880564` branch.
- Startup with app with `yarn start`
- Navigate to the `/profile` path to see details about a user displayed
- Change the `ID` supplied in the query to 1 or 3 or 4 and see the details change everytime.

#### Any background context you want to provide?
- This story is meant to kick in motion the development of the user profile page. There are other stories following on the heels of this one to flesh out and properly develop the profile page
- The `search` and `cart` components were disabled on certain pages because they only add to the noise on those pages. There's no need for them to be present. The pages are the `/login` and `/profile` pages. 

#### What are the relevant pivotal tracker stories?
- [#154880564](https://www.pivotaltracker.com/story/show/154880564)

#### Screenshots (if appropriate)
<img width="1280" alt="screen shot 2018-02-02 at 6 07 19 pm" src="https://user-images.githubusercontent.com/22825700/35745556-f2b20ade-0843-11e8-9689-dcccba43dc67.png">
